test/puppetlabs/puppetdb/command/dlo_test.clj

```diff
@@ -102,8 +102,8 @@
       (is (empty? (fs/list-dir dlo))))
 
     (testing "with subdirectories"
-      (let [subdir (fs/temp-dir dlo)
-            other-subdir (fs/temp-dir dlo)
+      (let [subdir (doto (fs/file dlo "dir-1") fs/mkdir)
+            other-subdir (doto (fs/file dlo "dir-2") fs/mkdir)
```

Comment:
@kbarber temp-dir If I recall correctly, temp-dir doesn't allow you to specify an absolute prefix (now?).  So in this case, you can't have a temp-dir in a temp-dir.  It throws an exception, but I'll double-check.